src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -647,6 +647,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
           </div>
         )}
 
+        <div className="toggle-room">
+          <h4 className="italic" style={{ fontSize: "20px" }}>
+            Show badges
+          </h4>
```

Comment:
Inline style objects like this create a new object on every render, which can cause extra performance things.

Since it appears to be a static value, could we handle this in a `.scss` class/etc instead?